prov/tcp/src/xnet_ep.c

```diff
@@ -156,6 +156,76 @@ static void xnet_set_no_port(SOCKET sock)
 #define xnet_set_no_port(sock)
 #endif
 
+int
+xnet_disable_keepalive(struct xnet_ep *ep)
```

Comment:
at least this function needs to be declared in xnet.h

Reply:
ah, missed that, thanks.